base/commands/serializer/validate.go

```diff
@@ -0,0 +1,93 @@
+package serializer
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/xeipuuv/gojsonschema"
```

Comment:
Could you remove the dependency to this library?
Later we may use another schema validator.